src/main/kotlin/no/nav/tiltaksarrangor/koordinator/service/KoordinatorService.kt

```diff
@@ -17,15 +17,43 @@ import no.nav.tiltaksarrangor.koordinator.model.MineDeltakerlister
 import no.nav.tiltaksarrangor.koordinator.model.TilgjengeligVeileder
 import no.nav.tiltaksarrangor.koordinator.model.VeilederFor
 import no.nav.tiltaksarrangor.model.StatusType
+import no.nav.tiltaksarrangor.model.exceptions.UnauthorizedException
+import no.nav.tiltaksarrangor.repositories.DeltakerlisteRepository
+import no.nav.tiltaksarrangor.repositories.model.DeltakerlisteDbo
+import no.nav.tiltaksarrangor.service.AnsattService
+import no.nav.tiltaksarrangor.utils.erPilot
+import no.nav.tiltaksarrangor.utils.isDev
 import org.springframework.stereotype.Component
 import java.util.UUID
 
 @Component
 class KoordinatorService(
-	private val amtTiltakClient: AmtTiltakClient
+	private val amtTiltakClient: AmtTiltakClient,
+	private val ansattService: AnsattService,
+	private val deltakerlisteRepository: DeltakerlisteRepository
 ) {
-	fun getMineDeltakerlister(): MineDeltakerlister {
-		return amtTiltakClient.getMineDeltakerlister().toMineDeltakerlister()
+	fun getMineDeltakerlister(personIdent: String): MineDeltakerlister {
+		val ansatt = ansattService.getAnsatt(personIdent) ?: throw UnauthorizedException("Ansatt finnes ikke")
+		val unikeRoller = ansatt.roller.map { it.rolle.name }.distinct()
```

Comment:
Kan vi ikke bare bruke `AnsattRolle` enumen direkte i stedet for å gjøre den om til String her og videre nedover?

Reply:
Ja, det bør jo gå an :)